coa-helm/values.yaml

```diff
@@ -77,3 +77,7 @@ workloadIdentity:
 
 postgres:
   podLocalDatabaseEnabled: true
+  podLocalDatabaseEnabledForTES: true
+  pgbouncer:
+    enabled: false

```

Comment:
Should this be `true` as per the comment "can be set to true in coa-helm" or is that a future thing?

Reply:
It should be false by default here, and will be set to true by Leo in cases where PgBouncer exists.